cleanlab_studio/internal/api/api.py

```diff
@@ -26,6 +26,7 @@
 
 try:
     import snowflake
+    import snowflake.snowpark as snowpark
```

Comment:
note I didn't verify this update will actually suffice to make the snowflake tutorial run properly.  But it should at least prevent all the other API tutorials from being broken like they currently are.

The snowflake tutorial is also broken in other ways:
https://github.com/cleanlab/cleanlab-studio-docs/issues/411

If you end up looking into it, please push other fixes for that tutorial too (and also push them for the databricks tutorial which has similar breakage).

Reply:
my guess is the v0.11 release of [snowflake package](https://pypi.org/project/snowflake/0.11.0/) on July 25 broke this import. 

So either: this needs to be updated to the new import with a lower-bound version of snowflake for our tutorial, or we upper-bound the version of snowflake to v0.10
our tutorial needs to version-cap snowflake, or best is if you find an easy fix that works for all snowflake versions